### DISCUSSION TITLE
How is Vanara compared to the WinApi project by @prasannavl?

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [dahall/Vanara#364] [Q&A] [answered]
https://github.com/prasannavl/WinApi

**Comment by dahall** [comment, marked as answer]
Both include a number of P/Invoke method and structure definitions from Win32. WinApi hasn't been updated in a while and contains far fewer methods. It does provide a nice very light-weight wrapper around the basic windows creation and message pump handling that is largely not included in Vanara along with some pretty useful wrappers to simplify fast graphics rendering in those windows. Much of the philosophy is similar. If you find tactics used there that you want to find in Vanara, but can't, please let me know.
